ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -238,22 +309,34 @@ - (void)callBroker:(ADAuthenticationCallback)completionBlock
     NSString* adalVersion = [ADLogger getAdalVersion];
     CHECK_FOR_NIL(adalVersion);
     
-    NSDictionary* queryDictionary = @{
-                                      @"authority": _context.authority,
-                                      @"resource" : _resource,
-                                      @"client_id": _clientId,
-                                      @"redirect_uri": _redirectUri,
-                                      @"username_type": _identifier ? [_identifier typeAsString] : @"",
-                                      @"username": _identifier.userId ? _identifier.userId : @"",
-                                      @"force" : _promptBehavior == AD_FORCE_PROMPT ? @"YES" : @"NO",
-                                      @"correlation_id": _correlationId,
+    NSDictionary* queryDictionary =
+    @{
+      @"authority"      : _context.authority,
+      @"resource"       : _resource,
+      @"client_id"      : _clientId,
+      @"redirect_uri"   : _redirectUri,
+      @"username_type"  : _identifier ? [_identifier typeAsString] : @"",
+      @"username"       : _identifier.userId ? _identifier.userId : @"",
+      @"force"          : _promptBehavior == AD_FORCE_PROMPT ? @"YES" : @"NO",
+      @"correlation_id" : _correlationId,
 #if TARGET_OS_IPHONE // Broker Message Encryption
-                                      @"broker_key": base64UrlKey,
+      @"broker_key"     : base64UrlKey,
 #endif // TARGET_OS_IPHONE Broker Message Encryption
-                                      @"client_version": adalVersion,
-									  BROKER_MAX_PROTOCOL_VERSION : @"2",
-                                      @"extra_qp": _queryParams ? _queryParams : @"",
-                                      };
+      @"client_version" : adalVersion,
+      BROKER_MAX_PROTOCOL_VERSION : @"2",
+      @"extra_qp"       : _queryParams ? _queryParams : @"",
+      };
+    
+    NSDictionary* resumeDictionary =
+  @{
+    @"authority"        : _context.authority,
+    @"resource"         : _resource,
+    @"client_id"        : _clientId,
+    @"redirect_uri"     : _redirectUri,
+    @"correlation_id"   : _correlationId,
+    };
+    
+    [[NSUserDefaults standardUserDefaults] setObject:resumeDictionary forKey:kAdalResumeDictionaryKey];
```

Comment:
@kpanwar Do we know what scenario we would even hit that codepath? It looks like it happens when we go through an auth code flow and instead get hit with "msauth://"--- the only way I could even think of hitting that is if the application isn't configured to use broker--- and yet the server decides to try and force it anyways? This doesn't seem like something we should be supporting.